pkg/util/client.go

```diff
@@ -117,3 +118,48 @@ func WatchFiles(candidates []string, callback func(fsnotify.Event)) error {
 
 	return nil
 }
+
+// LoadKubeConfigsV2 loads kubeconfigs. If the kubeconfigChangedCallBack is non-nil, it will watch all kubeconfigs it loaded
+// and call the callback once they change.
+func LoadKubeConfigsV2(opts flagutil.KubernetesOptions, kubeconfigChangedCallBack func()) (map[string]*rest.Config, error) {
+	configs, err := opts.LoadClusterConfigs(kubeconfigChangedCallBack)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load cluster configs: %w", err)
+	}
+	ret := map[string]*rest.Config{}
```

Comment:
Is this just a mapping from `rest.Config` to `*rest.Config`? Can we just make our downstream code expect `rest.Config` and get rid of `LoadKubeConfigsV2`, allowing us to simply use `opts.LoadClusterConfigs` ?

Reply:
Updated.